Library/Homebrew/utils/formatter.rb

```diff
@@ -32,7 +32,12 @@ def error(string, label: nil)
   end
 
   def wrap(s, width = 172)
-    s.gsub(/(.{1,#{width}})(\s+|\Z)/, "\\1\n")
+    col = 43
```

Comment:
why 43?

Reply:
This matches the value of `@desc_line_length` in `cli_parser.rb`, used when it wraps option descriptions for parser-based commands.

Reply:
Can you pull both into a constant? Thanks!